apollo-ios-codegen/Sources/ApolloCodegenLib/ApolloCodegen.swift

```diff
@@ -333,21 +401,21 @@ public class ApolloCodegen {
 
     nonFatalErrors.merge(
       try await nonFatalErrorCollectingTaskGroup() { group in
-        for graphQLObject in ir.schema.referencedTypes.objects {
+        for graphqlObject in ir.schema.referencedTypes.objects {
```

Comment:
We have always used `graphQL` across the project. I see why you want to change this, because it's hard to read and parse where the word `Object` begins and the word `graphQL` ends. It's just not the convention we've always been using. I'm happy with this change, but we should consider doing this going forward and changing old code as we come across it.

Pinging @calvincestari for visibility.

Reply:
So if I remember correctly there were some spots where this was done as `graphQL` and some where it was `graphql` so I just brought them all in sync with the one I thought to be more correct. Wasn't aware there was a standard in place for doing `graphQL` but open to switching back if we decide to do so. Agreed that if we keep this we need to ensure to update any other areas where this is the case.

Reply:
Yeah, there may have been some inconsistent places, but I have been using `graphQL`. But looking at this now, I think your way is an improvement on readability.

Reply:
I think @BobaFetters is right and it has been inconsistent. "GraphQL" is the whole name so I agree that lowercasing the entire thing feels more correct. We can make the change across the rest of the project in a separate PR though, it might just add noise to this one.